image/tools.cmds

```diff
@@ -38,13 +28,14 @@ sudo chmod a+x /opt/tools/chord/60.02_1.03/extractSigPredictHRD.R
 sudo chmod a+x /opt/tools/chord/2.00_1.14/extractSigPredictHRD.R
 sudo chmod a+x /opt/tools/chord/2.02_1.14/extractSigPredictHRD.R
 
-sudo chmod a+x /opt/tools/gridss/2.13.3/gridss
-sudo chmod a+x /opt/tools/gridss/2.13.3/gridsstools
-sudo chmod a+x /opt/tools/gridss/2.13.3/virusbreakend
-sudo chmod a+x /opt/tools/gridss/2.13.3/gridss_annotate_vcf_kraken2
-sudo chmod a+x /opt/tools/gridss/2.13.3/gridss_annotate_vcf_repeatmasker
+sudo chmod a+x /opt/tools/gridss/2.13.3-beta/gridss
+sudo chmod a+x /opt/tools/gridss/2.13.3-beta/gridsstools
+sudo chmod a+x /opt/tools/gridss/2.13.3-beta/virusbreakend
+sudo chmod a+x /opt/tools/gridss/2.13.3-beta/gridss_annotate_vcf_kraken2
+sudo chmod a+x /opt/tools/gridss/2.13.3-beta/gridss_annotate_vcf_repeatmasker
```

Comment:
Will improve this, the only reason for it is that `gridsstools` had to be recompiled and I didn't want to spend a bunch of time so made a new version. However it is only that one file that has been changed so I will make a copy of it in the "original" location and move it to the correct name at imaging time.